### PR TITLE
`namespace provisioning` using UI

### PR DESCRIPTION
**Pull request by droot**
This adds `namespace provisioning using UI` guide.

### CODE REVIEW

File: site/guides/namespace-provisioning-cli.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Namespace provisioning example via kpt CLI
+# Namespace provisioning using kpt CLI
```
Comment (yuwenma) [inline site/guides/namespace-provisioning-cli.md:1]: Do we want to align the naming between CLI and UI (UI is marked as "example" while CLI is not)

File: scripts/generate_site_sidebar/sidebar_template.md.tmpl [hunk 2]
```diff
@@ -56,7 +56,8 @@
   - [Contrib](https://catalog.kpt.dev/contrib/ ":target=_self")
 - [Guides](guides/)
   - [The Rationale behind kpt](guides/rationale.md)
-  - [Namespace provisioning example](guides/namespace-provisioning-example.md)
+  - [Namespace provisioning CLI](guides/namespace-provisioning-cli.md)
+  - [Namespace provisioning UI](guides/namespace-provisioning-ui.md)
```
Comment (yuwenma) [inline scripts/generate_site_sidebar/sidebar_template.md.tmpl:60]: I feel like removing the "example" could be confusing here：1. it seems to be the prerequisite to use Porch. 2. it's placed right after "Rationale" and before "Porch", which seems to be the second thing users should be consider (even before installing porch?)

File: site/guides/namespace-provisioning-ui.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
+# Namespace provisioning example using UI
+
+In this guide, we will use the package orchestration UI to:
+
+- Register blueprint and deployment repositories
```
Comment (yuwenma) [inline site/guides/namespace-provisioning-ui.md:5]: Suggest addling the link to tell what "blueprint", "deployment" and "repository" means here. It seems to be the term related to "repository".

File: site/guides/namespace-provisioning-ui.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+- Create a `kpt` package from scratch in blueprint repo
+- Create a deployable instance of the `kpt` package
```
Comment (yuwenma) [inline site/guides/namespace-provisioning-ui.md:7]: do we have a doc to talk about "variant constructor"?  It seems confusing especially for existing kpt users.

**Review by yuwenma — APPROVED** [review]

**Comment by yuwenma** [comment]
Approved since the comments are not blockers.

**Comment by droot** [comment]
Thanks Yuwen for the feedback. I have taken notes and will address them in the follow up PRs.

**Pull request merged**